modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -1677,9 +1673,4 @@ private boolean isTombstoneForCompaction(byte[] key, long revision) {
             );
         }
     }
-
-    private static WriteOptions createWriteOptions() {
-        // Metastorage recovery is based on the snapshot & external log. WAL is never used for recovery, and can be safely disabled.
-        return new WriteOptions().setDisableWAL(true);
```

Comment:
Why isn't WAL disabled anymore?

Reply:
Everything is okay now it’s in `org.apache.ignite.internal.metastorage.server.persistence.RocksDbKeyValueStorage#defaultWriteOptions`, it came from another PR.

```
    /** Metastorage recovery is based on the snapshot & external log. WAL is never used for recovery, and can be safely disabled. */
    private final WriteOptions defaultWriteOptions = new WriteOptions().setDisableWAL(true);
```